### PR TITLE
Edited story

### PR DESCRIPTION
**Pull request by Shane4666**
@casey-collab

### CODE REVIEW

File: english/wafflefries/wafflefries.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+You sit down at the local bar and find some fries, the fries are given. 
+with a marshmellow sauce!?!?!
+
+[Dip the fry in the marshmellow?](../sleep/marshmallow.md)
+[Watch TV at the bar](www.youtube.com)
```
Comment (casey-collab) [inline english/wafflefries/wafflefries.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

Comment (casey-collab) [inline english/wafflefries/wafflefries.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/wafflefries/wafflefries.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+You sit down at the local bar and find some fries, the fries are given.
```
Comment (casey-collab) [inline english/wafflefries/wafflefries.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Shane4666** [comment]
@casey-collab

File: english/wafflefries/wafflefries.md (hunk 2 above)
Comment (casey-collab) [inline english/wafflefries/wafflefries.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Shane4666** [comment]
@casey-collab

File: english/wafflefries/wafflefries.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
```
Comment (casey-collab) [inline english/wafflefries/wafflefries.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**